client/my-sites/upgrades/domain-search/domain-search.jsx

```diff
@@ -139,7 +134,7 @@ var DomainSearch = React.createClass( {
 							selectedSite={ selectedSite }
```

Comment:
That doesn't seem like a real case scenario, a user would not clear his local storage without reloading and expect the app to work.
We should however make sure that loading any page with an empty localStorage won't break this component. I'll do some testings